app/controllers/users_controller.rb

```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
```

Comment:
- [ ] Kindly add the views like wireframes and populate it with the data like in the [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#project-requirements)
     - add `index.html.erb` view to display user.photo, user.name and user.posts_counter.
     - Add also `app/views/users/show.html.erb` to show recent 3 posts.
     - **See all Post** links opens user's post index as displayed in wireframe
- [ ] Add views for posts like wireframe also:
     - Add `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page
     - In Post show page displays post with all comments listed as shown on the wireframe.